packages/shared/src/utils/helpers/dtrader-v2-helpers.ts

```diff
@@ -11,8 +11,13 @@ export const POSITIONS_V2_TAB_NAME = {
 
 export const getPositionsV2TabIndexFromURL = () => {
     const searchParams = new URLSearchParams(window.location.search);
```

Comment:
This issue is described in DTRA-2061 card. Briefly, for any language except EN, URL gas additional parameter 
`?lang=...` and that leads to wrong returned value from this function, because in such cases:
```
 const current_opened_tab = ['RU', 'open']; 
current_opened_tab[0] !== POSITIONS_V2_TAB_NAME.OPEN.toLowerCase().
```
I've added filtration for URL params and now this function is undependent from any langue change.